test/functional/inspec_exec_signed_profile_test.rb

```diff
@@ -0,0 +1,59 @@
+require "functional/helper"
+
+describe "inspec exec" do
+  include FunctionalHelper
+
+  def test_inspec_exec_signed_profile
+    prepare_examples do |dir|
+      skip_windows!
```

Comment:
Why are we skipping windows at this point?

Reply:
On windows the temporary file path wherein the test keys are getting generated is not getting resolved. This has no issues on Windows when tested manually.